### PR TITLE
More sync-up and one minor Makefile change

### PR DESCRIPTION
**Pull request by rodnymolina**

### CODE REVIEW

File: tests/helpers/docker.bash [hunk 1]
```diff
@@ -12,6 +12,10 @@ function wait_for_dockerd {
   retry_run 10 1 "docker ps"
 }
 
+function wait_for_dockerd {
```
Comment (ctalledo) [inline tests/helpers/docker.bash:15]: Looks like an error here (we are repeating an existing function).

**Review by ctalledo — COMMENTED** [review]
Caught a minor issue.

File: tests/helpers/docker.bash (hunk 1 above)
Comment (rodnymolina) [reply tests/helpers/docker.bash:15]: Interesting, not sure how that happened. Will generate a new patch once the CI concludes.

Comment (rodnymolina) [reply tests/helpers/docker.bash]: Done.

**Pull request merged**